src/compressed_tensors/quantization/quant_args.py

```diff
@@ -83,3 +83,32 @@ def get_observer(self):
         from compressed_tensors.quantization.observers.base import Observer
 
         return Observer.load_from_registry(self.observer, quantization_args=self)
+
+    @validator("strategy", pre=True)
+    def validate_strategy(cls, value, values):
```

Comment:
let's add in the following:
1. only setting `group_size=-1` or `strategy=channel` is needed to set to channel-wise
2. only setting `group_size=128` is sufficient for enabling grouping (if the group size is set to something non-default or not group then we raise an error)